src/main/scala/org/broadinstitute/dsde/firecloud/model/DataUse.scala

```diff
@@ -30,3 +31,26 @@ object DataUse {
   }
 
 }
+
+object ConsentCodes extends Enumeration {
+  val GRU = "GRU"
+  val HMB = "HMB"
+  val NCU = "NCU"
+  val NPU = "NPU"
+  val NMDS = "NMDS"
+  val NAGR = "NAGR"
+  val NCTRL = "NCTRL"
+  val RSPD = "RS-PD"
+  val IRB = "IRB"
+  val RSG = "RS-G"
+  val RSFM = "RS-FM"
+  val RSM = "RS-M"
+  val DSURL = "DS_URL"
+  val DS = "DS"
+
+  val booleanCodes = Seq(GRU, HMB, NCU, NPU, NMDS, NAGR, NCTRL, RSPD, IRB)
+  val genderCodes = Seq(RSG, RSFM, RSM)
+  val duRestrictionFieldNames = booleanCodes ++ genderCodes ++ Seq(DSURL/*, RSPOP*/)
+  val allDurFieldNames = duRestrictionFieldNames ++ Seq(DS)
+  val diseaseLabelsAttributeName: AttributeName = AttributeName.withLibraryNS(DS)
```

Comment:
Like the other one we were working with this afternoon, if this is used in only one place (or one class/file), it might be better living there. I'm thinking that based on the fact that this is an `AttributeName` which makes it opinionated about the namespace which probably isn't appropriate here.